css/fieldmanager-group-tabs.css

```diff
@@ -3,9 +3,33 @@
 }
 
 .fm-tab-bar .fm-tab {
-	border: 1px solid transparent;
+	border: 1px solid #dfdfdf;
 	border-bottom: none;
+	padding: 8px 10px;
+	border-radius: 3px 3px 0 0;
+	background: #f1f1f1;
+	margin-right: 5px;
```

Comment:
On reviewing this feature branch with FM Demos I noticed both vertical and horizontal tabs do not work well at mobile widths. Horizontal tabs break strangely and overlap, vertical tabs leave very little space for the actual tab content. Does FM offer mobile support in general?